# HG changeset patch
# User Rik <rik@octave.org>
# Date 1534197448 25200
#      Mon Aug 13 14:57:28 2018 -0700
# Node ID 842b6729d3ec76b6a09756002ee3b3ce1c4748f7
# Parent  0862570da0ae58fa179f30c7b0c76a673c98bc81
# Parent  e06ce02f4f74ae0284d20982a9deb92bced911a1
maint: merge stable to default.

diff --git a/doc/interpreter/genpropdoc.m b/doc/interpreter/genpropdoc.m
--- a/doc/interpreter/genpropdoc.m
+++ b/doc/interpreter/genpropdoc.m
@@ -800,16 +800,17 @@ box or \"out\".  __modemsg__.";
         s.doc = "Two-element vector @code{[2Dlen 3Dlen]} specifying the \
 length of the tickmarks relative to the longest visible axis.";
         s.valid = valid_2elvec;
 
       case "tightinset"
         s.doc = "Size of the @code{[left bottom right top]} margins \
 around the axes that enclose labels and title annotations.";
         s.valid = valid_4elvec;
+        s.printdefault = false;
 
       case "title"
         s.doc = "Graphics handle of the title text object.";
         s.valid = valid_handle;
 
       case "titlefontsizemultiplier"
         s.doc = "Ratio between the title fontsize and the tick \
 label fontsize";
@@ -1060,16 +1061,17 @@ z data.";
 
       case "editing"
         s.doc = doc_unused;
 
       case "extent"
         s.doc = "Vector @code{[x0 y0 width height]} indicating the size \
 and location of the text string.";
         s.valid = valid_4elvec;
+        s.printdefault = false;
 
       case "fontangle"
         s.doc = doc_fontangle;
 
       case "fontname"
         s.doc = doc_fontname;
         s.valid = valid_string;
 
@@ -1587,16 +1589,19 @@ point source (@qcode{\"local\"}).";
       ## Overridden shared properties
 
       ## Specific properties
       case "backgroundcolor"
       case "callback"
       case "cdata"
       case "enable"
       case "extent"
+        s.valid = valid_4elvec;
+        s.printdefault = false;
+
       case "fontangle"
         s.doc = doc_fontangle;
 
       case "fontname"
         s.doc = doc_fontname;
         s.valid = valid_string;
 
       case "fontsize"
